pkg/storage.go

```diff
@@ -112,6 +112,26 @@ type StoragePolicy struct {
 	MaxPools uint8
 }
 
+// Usage struct
+type Usage struct {
+	Size uint64
+	Used uint64
+}
+
+// Filesystem represents a storage space that can be used as a filesystem
+type Filesystem struct {
```

Comment:
Why do we suddenly have this type? Afaik in the past, a volume was identified just by its mountpoint. Why is this suddenly needed

Reply:
Because of the new `ListFilesystems() ([]Filesystem, error)`

Reply:
Yes, but why does that method return the new type then and not a list of mountpoints (strings)?

Reply:
Because that would mean you need to do some manual inspection of the volume from wherever this method is called. While now storaged does return you pretty much all the information you would want to know on a volume directly. I find this better abstract the logic of storaged then just returning a list of string.

We could change the type of the other method to use this new type too for consistency though.